packages/client-search/src/types/SearchResponse.ts

```diff
@@ -225,4 +225,6 @@ export type SearchResponse<TObject = {}> = {
    * The relevancy threshold applied to search in a virtual index.
    */
   appliedRelevancyStrictness?: number;
+
+  readonly renderingContent?: Settings['renderingContent'];
```

Comment:
Why is it `readonly` here? I feel like technically all the fields in this response is readonly, though.

Reply:
ah yes it shouldn't be, copied from the other place

Reply:
```suggestion
  renderingContent?: Settings['renderingContent'];
```